app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -57,6 +57,13 @@ def content_facet_ignore_update?(attributes)
             attributes['lifecycle_environment_id'].blank?
           )
         end
+
+        apipie :class do
```

Comment:
this now fails with 
```
NoMethodError: undefined method `apipie' for Katello::Concerns::ContentFacetHostExtensions:Module
/opt/theforeman/tfm/root/usr/share/gems/gems/katello-4.0.0.pre.master/app/models/katello/concerns/content_facet_host_extensions.rb:62:in `<module:ContentFacetHostExtensions>'
/opt/theforeman/tfm/root/usr/share/gems/gems/katello-4.0.0.pre.master/app/models/katello/concerns/content_facet_host_extensions.rb:3:in `<module:Concerns>'
/opt/theforeman/tfm/root/usr/share/gems/gems/katello-4.0.0.pre.master/app/models/katello/concerns/content_facet_host_extensions.rb:2:in `<module:Katello>'
/opt/theforeman/tfm/root/usr/share/gems/gems/katello-4.0.0.pre.master/app/models/katello/concerns/content_facet_host_extensions.rb:1:in `<top (required)>'
```

Reply:
that's strange.. I've run `apipie_dsl:cache` and it seems to be working:
![ScreenShot-1595420411004](https://user-images.githubusercontent.com/32508194/88175782-a195e400-cc26-11ea-9e66-4c8d1ef8898a.png)

Reply:
FTR adding the `extend ApipieDSL::Class` helped, however Oleh couldn't reproduce, maybe it's something on my side

Reply:
seems to be working for me, wondering if a 'spring stop' would have solved it?

Reply:
nah, this was prod-like env, anyway, if I was the only one, please ignore